src/allmydata/introducer/client.py

```diff
@@ -113,6 +114,39 @@ def connect_failed(failure):
         d = self._tub.getReference(self.introducer_furl)
         d.addErrback(connect_failed)
 
+    def load_announcements(self, storage_broker):
+        if os.path.exists(self.config_file):
+            f = open(self.config_file)
```

Comment:
``yaml.safe_load`` can return ``None`` if the file is empty, so we should have error reporting for this case.